src/Caching/Abstractions/ref/Microsoft.Extensions.Caching.Abstractions.csproj

```diff
@@ -1,10 +1,14 @@
 <!-- This file is automatically generated. -->
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netcoreapp3.0</TargetFrameworks>
```

Comment:
Spoke to @JunTaoLuo offline. We need to make sure this TFM works when we're source building netcoreapp3.1. I'd personally prefer not to cross-compile 3 different TFMs just for source build.

Reply:
I spoke with @crummel, turns out they are tracking it in https://github.com/dotnet/source-build/issues/759. Which is currently triaged for 3.1.